launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -45,12 +48,35 @@ def __init__(
         Most arguments are forwarded to :class:`launch_ros.actions.Node`, so see the documentation
         of that class for further details.
 
-        :param: node_name the name of the node, mandatory for full container node name resolution
-        :param: node_namespace the ros namespace for this Node, mandatory for full container node
-           name resolution
+        .. warning:: Parameters `node_name` and `node_namespace` are deprecated.
```

Comment:
@jacobperron BTW, it turns out there's a [`deprecated` directive](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) we can use.

Reply:
Using deprecated directive 50395ef
I wasn't sure what version number to put, so I used "Foxy".